### DISCUSSION TITLE
Possible to use web license with github pages.

### DISCUSSION THREAD

**Discussion by jordwalke** [fabrizioschiavi/pragmatapro#205] [Ideas]
Is it possible to use `PragmataPro` via github pages? Since `PragmataPro` is great for source code, I think a lot of people (myself included) would want to use it to showcase their open source projects which Github typically hosts. It would be good advertising for PragmataPro as well - however, I cannot see how to host `PragmataPro` on github pages in accordance with the license. Can you provide a suggestion?

**Comment by fabrizioschiavi** [comment]
Thanks Jordan! I agree with you. I would like to find a good way to showcase _PragmataPro_ in Github maybe as web font but it seems not so easy

**Comment by Tekl** [comment]
You could sell Github a huge site license. 🤣

**Comment by fabrizioschiavi** [comment]
Why not!  @github 😉
